test/wasm2js/emscripten.2asm.js.opt

```diff
@@ -201,10 +201,8 @@ var writeSegment = (
     }
   )(wasmMemory.buffer);
 writeSegment(1024, "aGVsbG8sIHdvcmxkIQoAAJwMAAAtKyAgIDBYMHgAKG51bGwp");
-writeSegment(1072, "EQAKABEREQAAAAAFAAAAAAAACQAAAAAL");
-writeSegment(1104, "EQAPChEREQMKBwABEwkLCwAACQYLAAALAAYRAAAAERER");
-writeSegment(1153, "Cw==");
-writeSegment(1162, "EQAKChEREQAKAAACAAkLAAAACQALAAAL");
+writeSegment(1072, "EQAKABEREQAAAAAFAAAAAAAACQAAAAALAAAAAAAAAAARAA8KERERAwoHAAETCQsLAAAJBgsAAAsABhEAAAARERE=");
+writeSegment(1153, "CwAAAAAAAAAAEQAKChEREQAKAAACAAkLAAAACQALAAAL");
```

Comment:
This change is due to a change in how the thesholds for merging are used. It used to be that if the number of zeroes was greater or equal to the threshold, they would be optimized out. Now the behavior is that the zeroes are only optimized out if their number is greater than the threshold. Restoring the old behavior and reverting the change to this test would certainly be possible, but would require tweaking almost all of the memory-packing tests, which is much more trouble than it's worth.